### PR TITLE
Add accelerator count to training history

### PR DESCRIPTION
**Pull request by ianstenbit**
This is relevant now because we're scaling learning rate and batch size based on the number of accelerators in #757

### CODE REVIEW

**Comment by LukeWood** [comment]
> /gcbrun

Gcb run is globally broken, but let’s merge as the tests are unaffected
